_data/links.yml

```diff
@@ -121,10 +121,12 @@
     
   - name: Map EPITA
     icon: fas fa-user-friends
-    href: http://map.epita.eu/
+    href: https://epimap.fr/
     class: tile-small-wide text-dark
     style:
-    - 'background-color: #fff'
+    - 'background-color: #230071'
+    - 'background-image: url(https://epimap.fr/img/logo-min.png)'
```

Comment:
The logo should not have borders, as they don't fit the item's tile.

Ideally, the logo should not have any background either.
The image should just contains the stylized brand name: `Epimap` in your case, with its icon.